src/app/components/Byline/utilities/bylineExtractor/index.tsx

```diff
@@ -55,7 +55,6 @@ const bylineExtractor = (blocks: object[]) => {
     originCode,
     locator,
     resolution: 160,
-    isPng: true,
```

Comment:
Removed this logic as I'm not sure it's need for the new ace/standard/ ichef URL?

Original URL: https://ichef.bbci.co.uk/news/png/160/cpsprodpb/f974/live/36226e20-94aa-11ec-9acc-37a09ce5ea88.png

Broken Link (replacing /news/ with /ace/standard/): https://ichef.bbci.co.uk/ace/standard/png/160/cpsprodpb/f974/live/36226e20-94aa-11ec-9acc-37a09ce5ea88.png

Working Link (replacing /news/png/ with /ace/standard/): 
https://ichef.bbci.co.uk/ace/standard/160/cpsprodpb/f974/live/36226e20-94aa-11ec-9acc-37a09ce5ea88.png